bugbug/models/duplicate.py

```diff
@@ -15,6 +15,9 @@
 from bugbug import bug_features, bugzilla, feature_cleanup, utils
 from bugbug.model import BugCoupleModel
 
+basicConfig(level=INFO)
+logger = getLogger(__name__)
```

Comment:
This is not necessary, it's already present a few lines after

Reply:
Makes sense, since it's the duplicate file =)

Gone ahead and fixed it